src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  background-color: #fff;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 60px;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    padding-top: 10px;
+    padding-left: 50px;
+  }
+
+  &__nav {
+    display: flex;
+    flex-direction: row;
+    padding-right: 50px;
+    margin-left: 243px;
+
+    @media (min-width: 1200px) {
+      & {
+        margin-left: 419px;
+      }
+    }
+
+    &-link {
+      @include smooth;
```

Comment:
Ensure that the `smooth` mixin is defined in your SCSS files. If it's not, you need to define it to avoid compilation errors.